src/qbindiff/loader/backend/utils.py

```diff
@@ -0,0 +1,426 @@
+# builtin import
```

Comment:
```suggestion
# builtin import
from __future__ import annotations
```

It will become the default in python 3.12 if I am correct but it's a good idea to always add it for the time being.